### PR TITLE
Add saving the log to a file via logback

### PR DESCRIPTION
**Pull request by dima-bzz**

### CODE REVIEW

File: log-viewer-cli/bin/logback.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<configuration scan="true" debug="false">
```
Comment (sevdokimov) [inline log-viewer-cli/bin/logback.xml:1]: `logback.xml` is a configuration file, not a binary. So it should be in the same folder as `config.conf`. You can either put `logback.xml` to the root, or move `logback.xml` and `config.conf` to "conf" directory.

**Comment by sevdokimov** [comment]
Sorry, I was wrong. :-)  `log-viewer-cli/bin` is a good location for `logback.xml`.  `log-viewer-cli/bin` contains files that will be packed into the archive. I was confused, I thought "/bin" is a directory **inside the archive**, not a directory in sources. Please, revert the last 2 commits. Probably `log-viewer-cli/bin` should be renamed to `log-viewer-cli/assets`

**Review by sevdokimov — APPROVED** [review]

**Comment by dima-bzz** [comment]
@sevdokimov Or maybe just download java and don't add it to $PATH. Therefore, it is very difficult to predict. I believe that we just provide a template, and users will customize themselves according to their requirements.

**Comment by sevdokimov** [comment]
> @sevdokimov Or maybe just download java and don't add it to $PATH. Therefore, it is very difficult to predict. I believe that we just provide a template, and users will customize themselves according to their requirements.

`/bin` directory is in $PATH , so calling `java` covers the case when we need to call `/bin/java`. Calling `java` covers more cases then `/bin/java`.

**Comment by dima-bzz** [comment]
@sevdokimov We can replace the absolute path with a relative one. But I did not check this solution, since java is not registered in my $PATH and I have the full path specified.

**Comment by dima-bzz** [comment]
@sevdokimov `/bin/java` I took from logviewer.sh

**Comment by dima-bzz** [comment]
@sevdokimov Alternatively we can make an installation script

File: log-viewer-cli/assets/logviewer.sh
Comment (sevdokimov) [inline log-viewer-cli/assets/logviewer.sh:1]: You didn't add `-Dlogback.configurationFile=./logback.xml` to logviewer.sh and logviewer.cmd. Forgot?

**Comment by dima-bzz** [comment]
@sevdokimov I thought that this is a console launch and there is no need for output to a file.

**Comment by sevdokimov** [comment]
> @sevdokimov I thought that this is a console launch and there is no need for output to a file.

Agree

**Review by sevdokimov — DISMISSED** [review]

**Comment by sevdokimov** [comment]
Please, resolve the conflicts.

**Review by sevdokimov — APPROVED** [review]

**Pull request merged**